electrumx/server/env.py

```diff
@@ -75,6 +75,10 @@ def __init__(self, coin=None):
         self.bandwidth_limit = self.integer('BANDWIDTH_LIMIT', 2000000)
         self.session_timeout = self.integer('SESSION_TIMEOUT', 600)
         self.drop_client = self.custom("DROP_CLIENT", None, re.compile)
+        self.blacklist_url = self.default('BLACKLIST_URL', None)
+        # temporary default
+        if self.blacklist_url is None and self.coin.NAME == 'BitcoinSegwit':
+            self.blacklist_url = 'https://electrum.org/blacklist.json'
```

Comment:
Would a `coins.py` member such as `BitcoinSegwit.FALLBACK_BLACKLIST_URL_FALLBACK` or whatever also handle this gracefully?